core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -795,6 +795,10 @@ class QualificationAppInfo(
     val sqlPlanInfoGraphEntry = SqlPlanInfoGraphBuffer.createEntry(sqlID, planInfo)
     checkMetadataForReadSchema(sqlPlanInfoGraphEntry)
     for (node <- sqlPlanInfoGraphEntry.sparkPlanGraph.allNodes) {
+      if (node.name.contains("Photon")) {
```

Comment:
Nit: Does Photon operators always begin with Photon keyword. If so can we use startsWith instead of contains here.

Reply:
I am not certain, but from previous observations in: https://github.com/NVIDIA/spark-rapids-tools/issues/449#issuecomment-1642339401, it looks like they begin with "Photon". I will update this.